docs/en/api/metrics-query-expression.md

```diff
@@ -87,6 +87,75 @@ The expression could be on the left or right side of the operator.
 | TIME_SERIES_VALUES      | SORTED_LIST/RECORD_LIST   | no     |                          |
 | SORTED_LIST/RECORD_LIST | SORTED_LIST/RECORD_LIST   | no     |                          |
 
+## Compare Operation
+Compare Operation takes two expressions and compares their results.
+The following table lists the compare operations supported by MQE.
```

Comment:
How does the comparison work for a labeled value?

Reply:
same as the binary op and the result is 0 or 1

Reply:
OK, so it would be values (0/1) with labels. How would you consider these labeled 0/1 fitting alerting?

Reply:
if the result has multiple labels, select the max matched times one
for example in 5 minutes, the percentil{_'P50,P75'} compare result is:
 P50(0,1,1,1,0), means match 3 times
 P75(0,1,0,1,0), means match 2 times
then the `matchCount` is 3

Reply:
Then when we implement `matchCount`, do we consider requiring a single time-series value or accepting one label?

Reply:
I see. So, we would still have multiple expected values to match multiple labels as before?

Reply:
no， the max matched label will be set to `matchCount`